scss/_input-group.scss

```diff
@@ -127,26 +127,34 @@
 // Remix the default form control sizing classes into new ones for easier
 // manipulation.
 
+.input-group-lg > .form-control:not(textarea),
+.input-group-lg > .custom-select {
```

Comment:
I don't think this would be needed for `.custom-select`, or is there a specific reason why you also added this here, @ysds?

Reply:
See #27677

Reply:
Hmm, it looks like my memory doesn't last for four days 😄